scss/mixins/_forms.scss

```diff
@@ -108,6 +117,16 @@
         & ~ button {
           border-color: $border-color;
         }
+
+        &:focus {
+          &[data-focus-visible-added] {
+            ~ button,
+            ~ .btn:disabled {
+              border-color: var(--#{$prefix}btn-border-color) !important; // stylelint-disable-line declaration-no-important
```

Comment:
This line is needed to override the grey border of the disabled button. Thanks to this line, we go from:
![lol](https://user-images.githubusercontent.com/92363071/231764439-fbe99730-a095-43e0-a92b-15ddf8def77b.png)

to:
![qs_error](https://user-images.githubusercontent.com/92363071/231764481-73e41a0b-fc14-4d95-a991-d7ad84e82ef3.png)

Please note that the button is still disabled even if the border is black on focus.

Reply:
I actually find it kind of weird to go from 
![image](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/91960143/31a39bbb-3761-4db5-a286-fec75ffc7cb0)

to 
![image](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/91960143/6c952dc2-81c7-4219-a699-79d1aae80e94)

The focus seems to disappear from the buttons 🤔 

Plus it seems that the `!important` isn't that important here and below.

Reply:
This question need to be addressed during next specs meeting.